src/Tasks/Microsoft.NET.Build.Tasks/GenerateDepsFile.cs

```diff
@@ -104,13 +109,43 @@ protected override void ExecuteCore()
                 PlatformLibraryName,
                 IsSelfContained);
 
+            Dictionary<PackageIdentity, StringBuilder> packagesThatWhereFiltered = null;
+
+            if (TargetManifestFileList != null && TargetManifestFileList.Length > 0)
+            {
+                packagesThatWhereFiltered = new Dictionary<PackageIdentity,StringBuilder>();
+
+                foreach (var targetManifestFile in TargetManifestFileList)
+                {
+                    Log.LogMessage(MessageImportance.Low, string.Format(CultureInfo.CurrentCulture, Strings.ParsingFiles, targetManifestFile));
+                    var packagesSpecified = StoreArtifactParser.Parse(targetManifestFile);
+                    var targetManifestFileName = Path.GetFileName(targetManifestFile);
+
+                    foreach (var pkg in packagesSpecified)
+                    {
+                        Log.LogMessage(MessageImportance.Low, string.Format(CultureInfo.CurrentCulture, Strings.PackageInfoLog, pkg.Id, pkg.Version));
+                        StringBuilder fileList;
+                        if (packagesThatWhereFiltered.TryGetValue(pkg, out fileList))
+                        {
+                            fileList.Append(" ");
```

Comment:
We should use a `;` separator here, so we can tell the difference between a file name with space in it, and two separate files.